make RIOT_CI_BUILD=1 CC_NOCOLOR=1 --no-print-directory -C /home/maribu/Repos/software/RIOT/tests/pbkdf2 test
r
socat - open:/dev/ttyACM3,b115200,echo=0,raw,cs8,parenb=0,cstopb=0 
READY
s
START
main(): This is RIOT! (Version: buildtest)
{ready}
passwd
{ready}
c2FsdA==
{ready}
1
VawEblbjCJ/sFpHCJUS2BflBhSFt3gRl5oudV8INrLw=

r
socat - open:/dev/ttyACM3,b115200,echo=0,raw,cs8,parenb=0,cstopb=0 
{ready}
r
{error}
{ready}
r
{ready}
r
{error}
{ready}
r
{ready}
Traceback (most recent call last):
  File "/home/maribu/Repos/software/RIOT/tests/pbkdf2/tests/02-random.py", line 80, in <module>
    test_base.main(VECTORS)
  File "/home/maribu/Repos/software/RIOT/tests/pbkdf2/tests/test_base.py", line 44, in main
    sys.exit(run(partial(test, vectors)))
  File "/home/maribu/Repos/software/RIOT/dist/pythonlibs/testrunner/__init__.py", line 27, in run
    child = setup_child(timeout, env=os.environ,
  File "/home/maribu/Repos/software/RIOT/dist/pythonlibs/testrunner/spawn.py", line 95, in setup_child
    sync_child(child, env)
  File "/home/maribu/Repos/software/RIOT/dist/pythonlibs/testrunner/spawn.py", line 132, in sync_child
    utils.test_utils_interactive_sync(child,
  File "/home/maribu/Repos/software/RIOT/dist/pythonlibs/testrunner/utils.py", line 30, in test_utils_interactive_sync
    _test_utils_interactive_sync(child, retries, delay)
  File "/home/maribu/Repos/software/RIOT/dist/pythonlibs/testrunner/utils.py", line 22, in _test_utils_interactive_sync
    child.expect_exact(ready_exp, timeout=0)
  File "/usr/lib/python3.10/site-packages/pexpect/spawnbase.py", line 421, in expect_exact
    return exp.expect_loop(timeout)
  File "/usr/lib/python3.10/site-packages/pexpect/expect.py", line 181, in expect_loop
    return self.timeout(e)
  File "/usr/lib/python3.10/site-packages/pexpect/expect.py", line 144, in timeout
    raise exc
pexpect.exceptions.TIMEOUT: Timeout exceeded.
<pexpect.pty_spawn.spawn object at 0x7ff05db699f0>
command: /usr/bin/make
args: [b'/usr/bin/make', b'cleanterm']
buffer (last 100 chars): 'dy}\r\n'
before (last 100 chars): '0,echo=0,raw,cs8,parenb=0,cstopb=0 \r\n{ready}\r\n{error}\r\n{ready}\r\n{ready}\r\n{error}\r\n{ready}\r\n{ready}\r\n'
after: <class 'pexpect.exceptions.TIMEOUT'>
match: None
match_index: None
exitstatus: None
flag_eof: False
pid: 27730
child_fd: 5
closed: False
timeout: 10
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0.05
delayafterclose: 0.1
delayafterterminate: 0.1
searcher: searcher_string:
    0: 'READY'
make: *** [/home/maribu/Repos/software/RIOT/makefiles/tests/tests.inc.mk:22: test] Error 1

Return value: 2
